core/build.gradle.kts

```diff
@@ -19,50 +16,46 @@ configurations.all {
 dependencies {
     implementation(project(":grpc-read-db"))
 
-    implementation("com.exactpro.th2:common:5.10.0-dev")
-    implementation("com.exactpro.th2:common-utils:2.2.2-dev")
-    implementation("com.exactpro.th2:lw-data-provider-utils:0.0.1-dev")
+    implementation(libs.th2.common)
+    implementation(libs.th2.common.utils)
+    implementation(libs.th2.lw.data.provider.utils)
 
     implementation("org.slf4j:slf4j-api")
```

Comment:
note: Why didn't you move this dependency to libs catalog? Is it special somehow?

Reply:
I moved all dependency with explicit version to catalog to be able to look at them in one place and change in one place